### PR TITLE
feat(admin): Allow support agents to send password reset email

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- This feature was requested by our support team

## This pull request

- Allows a support agent to iniitate the password reset flow for a user
- Reuses the fxa-auth-client to call FxA api
- Adds nestjs backend module (looking to refactor this since it exisits in fxa-graphql-api project too)
- Adds guards for the feature

## Issue that this pull request solves

Closes: https://mozilla-hub.atlassian.net/browse/FXA-5716

## Checklist

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots

Screen Shot 2022-09-13 at 1 36 08 PM

### CODE REVIEW

File: packages/fxa-admin-panel/src/components/AccountSearch/Account/index.test.tsx [hunk 1]
```diff
@@ -218,3 +218,12 @@ it('displays the locale', async () => {
   expect(getByTestId('account-locale')).toHaveTextContent('en-US');
   expect(getByTestId('edit-account-locale')).toBeInTheDocument();
 });
+
+it('displays sends password reset', async () => {
```
Comment (dschom) [inline packages/fxa-admin-panel/src/components/AccountSearch/Account/index.test.tsx:222]: Should it be 'send' instead of 'sends'.

Comment (dschom) [inline packages/fxa-admin-panel/src/components/AccountSearch/Account/index.test.tsx:222]: Should this be 'send' instead of 'sends'?

**Review by dschom — APPROVED** [review]
LGTM!

**Review by chenba — APPROVED** [review]
👍 LGTM but I didn't try it locally

**Pull request merged**